src/import-service/serverless.yml

```diff
@@ -0,0 +1,37 @@
+service: ImportService
+
+frameworkVersion: '3'
+
+provider:
+  name: aws
+  runtime: nodejs14.x
+  region: eu-central-1
+  httpApi:
+    cors: true
+  iamRoleStatements:
+    - Effect: Allow
+      Action: s3:*
+      Resource:
+        - arn:aws:s3:::cloudx-aws-practitioner-js
+    - Effect: Allow
+      Action: s3:*
+      Resource:
+        - arn:aws:s3:::cloudx-aws-practitioner-js/*
+
+functions:
+  importProductsFile:
+    handler: importProductsFile.handler
+    events:
```

Comment:
In the task description we have this: "The **name** will be passed in a query string as a **name parameter** and **should** **be** **described** in the serverless.yml file **as a request parameter.**" I use serverless.ts instead of serverless.yml and can't give you any advice on where to put this parameter but I don't see it here. Not sure if this is 100% critical but still...